sftpgo-plugin-auth.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: sftpgo-plugin-auth
+  version: 1.0.8
+  epoch: 0
+  description: "LDAP/Active Directory authentication for SFTPGo"
+  copyright:
+    - license: AGPL-3.0-only
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - git
+      - go
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/sftpgo/sftpgo-plugin-auth
+      tag: v${{package.version}}
+      expected-commit: 1513afda581921cb504be44ad2191db47504f7f9
+
+  - uses: go/bump
+    with:
+      deps: google.golang.org/grpc@v1.64.1
+
+  - uses: go/build
+    with:
+      packages: .
+      output: sftpgo-plugin-auth
+      ldflags: -s -w -X github.com/sftpgo/sftpgo-plugin-auth/cmd.commitHash=$(git describe --always) -X github.com/sftpgo/sftpgo-plugin-auth/cmd.buildDate=$(date -u +%FT%TZ)
```

Comment:
to support govulnchecker in full we typically build without "-s". Also the go/build pipeline automatically specifies "-w" already.

Thus "-s -w" should be dropped from this and all other files.

I guess I really must make that a failure in the go/build pipeline.

Reply:
thanks !